migrations/2024-09-16-174833_create_oauth_provider/up.sql

```diff
@@ -14,7 +14,7 @@ CREATE TABLE oauth_provider (
     scopes text NOT NULL,
     auto_verify_email boolean DEFAULT TRUE NOT NULL,
     account_linking_enabled boolean DEFAULT FALSE NOT NULL,
-    enabled boolean DEFAULT FALSE NOT NULL,
+    enabled boolean DEFAULT TRUE NOT NULL,
```

Comment:
I wonder if there's some reason this was default false, or if it was just an accident. @privacyguard